recipes-utils/ultra96-ap-setup/ultra96-ap-setup_%.bbappend

```diff
@@ -2,18 +2,4 @@ LIC_FILES_CHKSUM = "file://wpa_ap.conf;md5=24ab6a95620be06cef908de36a5d01c3"
 
 FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
-SRC_URI:remove = " \
-    file://ap.sh \
-    "
-
 INITSCRIPT_PARAMS = "start 99 S . stop 90 6 . stop 90 0 ."
```

Comment:
Do you still need this now as the service is now systemd and therefore I don't think you'll still need the SystemV init install params.

Reply:
The way Xilinx modified this recipe allows users to use this with SystemV init or systemd.
That's why I left the INITSCRIPT_PARAMS. I tried it with a systemV init and it was still working fine.